core/domain/config_domain.py

```diff
@@ -304,6 +304,15 @@ def get_all_config_property_names(cls):
     'promo_bar_message', UNICODE_SCHEMA,
     'The message to show to all users if the promo bar is enabled', '')
 
+COMMUNITY_MESSAGE_ENABLED = ConfigProperty(
```

Comment:
Is there a way to control the order in which the configs appear in the admin dashboard? In the screencast, the boolean flag config appeared two configs below the message config. Can we group them together?

Reply:
Yeah I was too confused and realized that it appears in a random order (basically it fetches all the config property in a dict and renders them accordingly)

Reply:
I think there isn't, currently -- this was an oversight when we were initially writing the config property code. It would probably be a bigger change outside the scope of this PR...

Reply:
Ah, gotcha. Maybe add a comment explaining this and create an issue to refactor this for the future (if the comment and issue don't already exist)?